src/BenchmarkDotNet/EventHandlers/BenchmarkEventHandlerBase.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains.Results;
+using BenchmarkDotNet.Validators;
+
+namespace BenchmarkDotNet.EventHandlers
+{
+    public abstract class BenchmarkEventHandlerBase
```

Comment:
this is very subjective, but `EventProcessor` suggested by @bartonjs sounds best to me.

```suggestion
    public abstract class EventProcessor
```